app/controllers/InstanceRegistryController.scala

```diff
@@ -44,11 +52,17 @@ class MyExecutionContextImpl @Inject()(system: ActorSystem)
   * @param controllerComponents
   * @param ws
   */
-class InstanceRegistryController @Inject()(myExecutionContext: MyExecutionContext,
+
+
+class InstanceRegistryController @Inject()(implicit system: ActorSystem, mat: Materializer, myExecutionContext: MyExecutionContext,
                                            val controllerComponents: ControllerComponents,
                                            ws: WSClient, config: Configuration)
   extends BaseController {
+
+
+  var pubActor: ActorRef = null
```

Comment:
Please use `Option` instead of `null` throughout

Reply:
This was addressed during the refactoring of the socket connection to the instance registry ( code was moved inside the publish socket actor, which is now a lazy val)